### ISSUE TITLE
Use ontology access kit to identify environmental terms for each triad

### ISSUE THREAD

**Issue by mslarae13** [microbiomedata/issues#843]
use ontology access kit for a LLM approach to identifying the environmental terms that are fitting for each extension

https://incatools.github.io/ontology-access-kit/index.html  

Apply this LLM method to all extensions

MixS Environment name | submission portal DhInterface name | harmonizerApi.ts status | priority in https://github.com/microbiomedata/issues/issues/468#issuecomment-2243964338 | env_broad_scale | env_local_scale | env_medium
-- | -- | -- | -- | -- | -- | --
PlantAssociated | PlantAssociatedInterface | published | high |   |   |  
Sediment | SedimentInterface | published | high |   |   |  
Soil | SoilInterface | published | high |   |   |  
Water | WaterInterface | published | high |   |   |  
Air | AirInterface | published | low |   |   |  
BuiltEnvironment | BuiltEnvInterface | published | low |   |   |  
HostAssociated | HostAssociatedInterface | published | low |   |   |  
HydrocarbonResourcesCores | HcrCoresInterface | published | low |   |   |  
HydrocarbonResourcesFluidsSwabs | HcrFluidsSwabsInterface | published | low |   |   |  
MicrobialMatBiofilm | BiofilmInterface | published | low |   |   |  
MiscellaneousNaturalOrArtificialEnvironment | MiscEnvsInterface | published | low |   |   |

**Comment by mslarae13** [comment]
Break into 3 smaller tasks for @turbomam , @Natalie-Winans , & @sierra-moxon to divide & concur

Remove LLM for now. Separate issue!

Run the queries we decided policy on, adjust as glaringly needed/ must do.

Update this table!

**Comment by mslarae13** [comment]
@turbomam I think it's safe to say this is in progress?
And you're getting lists into google drive/ sheets for use to review. Yes?

**Comment by turbomam** [comment]
I got the sense that this issue was about designing OAK queries for all of the high priority environments, and that the issues below were specifically about Soil. 

Closing it is fine with me if you think the all-environments aspect is captured somewhere else.

I also really like that prioritization and cross-reference table at the top and hope we find a more prominent home for it.

- https://github.com/microbiomedata/issues/issues/849
- https://github.com/microbiomedata/issues/issues/848
- https://github.com/microbiomedata/issues/issues/847
